src/components/sections/auth/OAuthButtons.tsx

```diff
@@ -0,0 +1,54 @@
+'use client'
+
+import Image from 'next/image'
+import {signIn} from 'next-auth/react'
+
+import twitterLogo from '@public/images/oauth/twitter-logo.svg'
+import googleLogo from '@public/images/oauth/google-logo.svg'
+import githubLogo from '@public/images/oauth/github-logo.svg'
+import {PATHS} from '@/lib/utils/common/paths'
+
+const OAuthButtons = () => {
+  return (
+    <div className="flex flex-col gap-6">
+      <button
+        onClick={() => signIn('google', {callbackUrl: PATHS.myProjects})}
+        className="flex h-12 w-96 items-center justify-center bg-white">
+        <Image
+          src={googleLogo as string}
+          alt="Google logo"
+          className="h-6 w-6"
+          width={24}
+          height={24}
+        />
+        <span className="pl-2 font-bold text-black">Sign Up with Google</span>
```

Comment:
I think it should be "Sign in with Google", "Sign up" doesn't really make sense.

Reply:
You're right, just followed figma on this one 😅